### DISCUSSION TITLE
Getting the '╱' character to tile nicely?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [wezterm/wezterm#3780] [Q&A] [answered]
I've got this in my neovim config:

```lua
vim.opt.fillchars = "diff:╱"
```

In kitty, it tiles nicely, but looks rather weird in wezterm. Does anyone have any ideas on whether I can fix this through the config?

I'm using the JetBrainsMono Nerd Font.

## Screenshots

![Screenshot 2023-05-26 at 15 18 03](https://github.com/wez/wezterm/assets/994357/40f09baf-bcfa-44f4-9cce-499c6a04fa26)

Closeup in wezterm:

![Screenshot 2023-05-26 at 15 20 52](https://github.com/wez/wezterm/assets/994357/98f05317-e3da-42d0-bb68-1004c8dae0e7)

**Comment by wez** [comment]
What does `wezterm ls-fonts --text ╱ --rasterize-ascii` output for you?

**Reply by gegoune** [reply]
Not an OP, but facing the same issue, here is mine with Iosevka:
<img width="1334" alt="slash" src="https://github.com/wez/wezterm/assets/69750637/d40b6778-330d-4bdc-b024-48652b834480">

**Reply by fredrikaverpil** [reply]
@wez sure, see below. Also, here's my [wezterm.lua](https://github.com/fredrikaverpil/dotfiles/blob/main/wezterm.lua).

![Screenshot 2023-05-27 at 09 00 51](https://github.com/wez/wezterm/assets/994357/458b1d65-ffe7-415c-aca8-ee3d19503356)

**Reply by wez** [reply, marked as answer]
The rasterization from the custom glyph looks OK, so the issue must be at a different layer. I'd encourage you to file a proper bug report if you want to go further with this.

**Reply by fredrikaverpil** [reply]
@wez @gegoune I filed a bug: https://github.com/wez/wezterm/issues/3788
